posts/Andrea/2019-07-29-hibernate-orm-544-final-release.adoc

```diff
@@ -0,0 +1,32 @@
+= "Hibernate ORM 5.4.4.Final released"
+Andrea Boriero
+:awestruct-tags: [ "Hibernate ORM", "Releases" ]
+:awestruct-layout: blog-post
+:released-version: 5.4.4.Final
+:release-id: 31774
+---
+
+We just released the fourth maintenance release of Hibernate ORM 5.4.
+
+It is designed to be a drop in replacement for ORM 5.4.3.
+
+== What's new
```

Comment:
We have a huge amount of performance optimisations in this release. Not suggesting to mention them all, but at least put the spotlight on performance?

e.g. I would mention it also in the title:

> Hibernate ORM 5.4.4.Final released with significant performance improvements

Reply:
Personal preference would be to not add this to the title.  But I'm fine either way

Reply:
Also, there also are some bug-fixes :)  So we should mention that.  To a degree people equate maintenance release with bug-fix so maybe it is not important to explicitly highlight that.  But I would.

I'd also suggest just getting rid of the "What's new" and "Bugfixes" section headers.  I discriminate against sections with a single sentence haha

Something like:

````
We just released the fourth maintenance release of Hibernate ORM 5.4 which includes significant performance improvements in addition to bugfixes.  

There are a number of changes that cumulatively lead to the performance improvements, the major one being the addition of a new feature to use http://in.relation.to/2019/07/29/bytecode-proxy[bytecode enhancement as a proxy].

You can find the full list of changes ...

== Getting 5.4.4.Final
...
````